coding-concepts.js

```diff
@@ -6,54 +6,65 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+5
+
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
+// a) Your answer: 5
 // b) Verify and explain:
-
+// My answer was correct. The code that is currently written as is, would output the number of 5. Currently, there are four different elements in the array. However, we can see that there is a .push() built-in being used, it means that we are asking the computer to add "indigo" into the array. As we used the function console.log() and built-in method .push() to add in "indigo" as another element, the output of this, will give us the amount of the array, including the newest element. This in turn will change our array amount from 4 to 5. 
 
 // --------------------1) What will this log?
+10
 
-const cohort = "LEARN 2022"
+// const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
+// a) Your answer: 10 
 // b) Verify and explain:
+// My answer was correct as I predicted the output would come out as 10. Looking at this, I can see that the variable is being assigned a data type of a string with the information of "LEARN 2022". I can see there is a .length within our code. .length() is a informational command that returns the number of elements in the array. By utilizing function console.log(), we are asking the computer to recall the declared variable cohort and use the built-in method .length to output the amount of elements used within the string. "LEARN 2022" has an index amount of 9. Because .length counts the elements from 0, it adds 1 to the last index. This is why I knew the output would be 10. 
```

Comment:
just to add some clarity. `.length` will count all the values in the array or string. Values in an array are separated by commas. Values in a string are all the characters including spaces within a set of quotation marks.